src/app/(public)/settings/page.tsx

```diff
@@ -0,0 +1,18 @@
+import ProfileSettings from '@/components/profile-settings/profile-settings'
+import { getUser } from '@/lib/auth/lucia'
+import { getUserProfile } from '@/services/actions/userActions'
+import { redirect } from 'next/navigation'
+import React from 'react'
+async function page() {
+  const sesion = await getUser()
+  if (!sesion?.id) {
+    redirect('/auth')
+  }
+  const user = await getUserProfile()
```

Comment:
no need for 

```
const sesion = await getUser()
  if (!sesion?.id) {
    redirect('/auth')
  }
```

this is because getUserProfile already checks for this